### DISCUSSION TITLE
Git and GitHub Workflow | 3rd Issue Ticket Task | Working with GitHub web UI

### DISCUSSION THREAD

**Discussion by Ifycode** [collabo-community/.github#2] [Git and GitHub Workflow "Good First Issue" Tickets]
#### Prerequisite

This 3rd task issue ticket is based on the 1st - 2nd issue tickets in this Git and GitHub workflow series. To proceed with this 3rd task, ensure that you have submitted pull requests for all the previous tasks, and your pull requests for all the previous tasks have been approved.

If you have not done any or all of the previous tasks, you can find them through the project board here: [git and github workflow good first issue tickets](https://github.com/orgs/collabocate-osc/projects/1?filterQuery=label%3A%22For%3A+Community+%5BGeneral%5D%22+label%3A%22topic%3A+good+first+issue%22+label%3A%22topic%3A+git+%26+github+workflow%22)

If your pull requests for all the previous tasks have been submitted and approved by a maintainer, you can proceed to do this 3rd task.

#

#### Helpful resource
This youTube playlist will give you an idea of how to work with GitHub web User Interface: https://youtube.com/playlist?list=PLMDhbo3xlD1ESa1_9WN4yG7bGDDksEQx7&si=F9lzacG1cr9WGS-M

#

#### Issue description

Well done, and nice work so far!
In this **3rd task assignment**, you are required to rename the markdown tutorial file that you created in the one of the previous tasks **from the GitHub web User Interface**. Find more information in the expected solution section below.

#

#### Expected solution

> [!TIP]  
> You are to sync your fork, create your branch, and make the changes to the already existing files or folders using the GitHub web User Interface.

- You may have to first sync your fork's `develop` branch with the @collabocate-osc repo's `develop` branch, so that your previous tasks' changes that we merged into our develop branch can reflect in your fork's develop branch.
- Create a new branch from the `develop` branch. Your branch name should be `@GH-replaceThisPartAfterTheHyphenWithTheIssueNumberForThisTicket` (don't add the # sign to the ticket number, and no space in there). It is in this new branch that you should make your changes in and submit from.
- Rename the`gitcollab-ui-task.md` file that you previously created to `gitcollab-web-ui-task.md`

#

#### Additional context
- Let us know in the comment of this issue ticket (GitHub discussion) that you will be participating in this 3rd task assignment. Ensure to tag @collabocate-osc/maintainers in your comment. That way, maintainers will be notified and will be prepared to help review your pull request when you submit.
- Repo URL (where you are to do and submit this task to): https://github.com/collabocate-osc/good-first-issue-GIT-solutions
- If you have any questions or need any clarification when working on this ticket, please use the `#git-workflow-support` channel in our [discord community server](https://resources.collabocate.community).
- To help you better understand how to work with other contributors within @collabocate-osc better, we will review your pull request based on the directions given in this issue ticket and our [Public Documentation Resource Guide](https://resources.collabocate.community).

#

#### Testing checklist

Note (to contributor taking on this task assignment): Add the testing checklist below (without this note) to your pull request when submitting your fix. This will guide (you and) the contributors who will review your fix to know what is important to check or test for.
- [ ] Here's a link to my approved 2nd task pull request: `Replace this inline code block with the link to your approved 2nd task pull request`
- [ ] My branch for this task assignment is created from the `develop` branch, and the name of my branch is a mix of `@GH-` and the ticket's issue number (no # sign and no space in there).
- [ ] I have renamed the`gitcollab-ui-task.md` file in my own folder to `gitcollab-web-ui-task.md`
- [ ] I certify that I ran my checklist.

**Comment by Goodnews-code** [comment]
Hi @collabo-community/maintainers, i will be participating in this 3rd assignment task.

**Reply by Ifycode** [reply]
👍🏼

**Comment by BA-atomic** [comment]
Hi @collabo-community/maintainers, i will be participating in this 3rd assignment task.

**Reply by Ifycode** [reply]
👍🏼

**Comment by OpeyemiAdeniji** [comment]
Hi @collabo-community/maintainers, i will be participating in this 3rd assignment task.

**Reply by Ifycode** [reply]
👍🏼

**Comment by Oyebliss** [comment]
Hi @collabo-community/maintainers, i will be participating in this 3rd assignment task.

**Reply by Ifycode** [reply]
👍🏼

**Comment by KBandipo** [comment]
Hi @collabo-community/maintainers 
I will be participating in this 3rd task assignment.

**Reply by Ifycode** [reply]
👍🏼

**Comment by Danbaba1** [comment]
Hi @collabo-community/maintainers
I will be participating in this 3rd task assignment.

**Reply by Ifycode** [reply]
👍🏼

**Comment by elizabethobiagba** [comment]
Hi @collabo-community/maintainers , I will be participating in this 3rd task assignment

**Reply by Ifycode** [reply]
👍🏼

**Comment by oluwadamilola21** [comment]
Hi @collabo-community/maintainers, i will be participating in this 3rd assignment task.

**Reply by elizabethobiagba** [reply]
Do go right ahead 👍🏽

**Comment by Ifycode** [comment]
> [!IMPORTANT]  
> The name of our community has changed from `collabo-community` to `collabocate-osc`. Please tag maintainers with @collabocate-osc/maintainers from now on. Thank you.

**Comment by Yanmi01** [comment]
Hi @collabocate-osc/maintainers, I'll be participating in this 3rd assignment task

**Reply by Ifycode** [reply]
👍🏼
